apps/src/templates/instructions/teacherFeedback/Rubric.jsx

```diff
@@ -27,14 +26,19 @@ class TeacherFeedbackRubric extends Component {
     const {
       rubric,
       performance,
-      isReadonly,
-      disabledMode,
+      isEditable,
       onRubricChange,
       viewAs
     } = this.props;
 
-    const showFeedbackInputAreas =
-      !isReadonly && (!!performance || viewAs === ViewType.Teacher);
+    let showFeedbackInputAreas, expandByDefault;
```

Comment:
Would this be clearer if renamed `expandByDefault` to `expandAllByDefault` or `expandAllRubricFields`?  And maybe add some comments that explain the various cases?  (I think part of the problem is also that I find `performance` to be a confusing name but we can't really change it.)